### PR TITLE
Query results on Ceph based on date

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: #2272

## This introduces a breaking change

- [x] No

## This should yield a new module release

- [x] No

### CODE REVIEW

File: thoth/storages/result_base.py [hunk 1]
```diff
@@ -84,12 +86,38 @@ def connect(self) -> None:
...
+        if end_date is None:
+            end_date = date.today() + timedelta(days=1)  # Today inclusively.
+
+        walker = start_date
+        step = timedelta(days=1)
+        while walker < end_date:
+            yield walker.strftime(f"{self.RESULT_TYPE}-%y%m%d")
+            walker += step
+
+    def get_document_listing(
```
Comment (pacospace) [inline thoth/storages/result_base.py:102]: missing `get_document_count` to have the new parameters

**Review by pacospace — COMMENTED** [review]
One comment, thanks @fridex !!

File: thoth/storages/result_base.py (hunk 1 above)
Comment (fridex) [reply thoth/storages/result_base.py:104]: Add, PTAL 👍🏻

**Review by pacospace — APPROVED** [review]
Thanks 💯 🥇 works magnificiently!

**Pull request merged**